pages/studentPortal/messageRepository/index.tsx

```diff
@@ -1,25 +1,28 @@
 import { useMutation } from "@apollo/client";
 import { format } from "date-fns";
-import React, { useState } from "react";
+import { useState } from "react";
+import MessageFeed from "../../../components/studentPortal/messageRepository/MessageFeed";
 import { Button } from "../../../components/ui/Button";
+import { FETCH_ALL_MESSAGES } from "../../../graphql/studentPortal/messageRepository/fetchMessages";
 import { insert_Message } from "../../../graphql/studentPortal/messageRepository/insertMessage";
-import { useAuth } from "../../../lib/authContext";
 type Message = {
   message: string;
   date: string;
 };
 function MessageRepository() {
   // Initialize state for the message input.
   const [message, setMessage] = useState("");
-  const { user } = useAuth();
+  // const { user } = useAuth();
 
   //date for state var
   const [date, setDate] = useState(new Date());
 
   const [targetDate, setTargetDate] = useState({});
 
   // route back to goals overview page on complete
-  const [saveNewMessage] = useMutation(insert_Message);
+  const [saveNewMessage] = useMutation(insert_Message, {
+    refetchQueries: [{ query: FETCH_ALL_MESSAGES }],
```

Comment:
this isn't working, can we figure this out, or just merge in for now and fix later?